src/boot/efi/boot.c

```diff
@@ -1185,6 +1189,26 @@ static BootEntry* boot_entry_free(BootEntry *entry) {
 
 DEFINE_TRIVIAL_CLEANUP_FUNC(BootEntry *, boot_entry_free);
 
+static bool parse_console_color(const char *v, uint8_t *color_out){
+        const char *color_names[] = {
```

Comment:
the array would be in writable memory, this needs a 2nd const. Also, should be static.

the colors are actually densly backed afaics, hence the color values should just be the index into this array, i.e.

```c
static const char *const color_names[] = {
        [EFI_BLACK] = "black",
        …
}
```

you get the idea.